src/store/flow/flow.ts

```diff
@@ -16,20 +17,69 @@ import moment from 'moment'
 import { ActionTree, GetterTree, MutationTree } from 'vuex'
 import { IRootState } from '@/store'
 import {
-  defaults, includes, forEach, cloneDeep, get, has,
+  defaults, 
+  includes, 
+  forEach, 
+  cloneDeep, 
+  get, 
+  has, 
+  omit
 } from 'lodash'
 import { discoverContentTypesFor } from '@/store/flow/resource'
 import { computeBlockPositionsFrom } from '@/store/builder'
 import { IFlowsState } from '.'
 
 export const getters: GetterTree<IFlowsState, IRootState> = {
-  activeFlow: (state) => state.flows.length && getActiveFlowFrom(state as unknown as IContext),
-
+  //We allow for an attempt to get a flow which doesn't yet exist in the state - e.g. the firstFlowId doesn't correspond to a flow
+  activeFlow: (state) => {
+    if(state.flows.length) { 
+      try {
+        return getActiveFlowFrom(state as unknown as IContext)
+      } catch(err) {
+        return 0
```

Comment:
This was me trying to preserve the original type I was finding on master to avoid any weirdness but let me just try and switch to a bool and see how it goes. I think QA will pick up any issues I miss.